ci-scripts/install-travis-gcc.sh

```diff
@@ -15,6 +17,8 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
   git show 42d31bba7772fb01f9ba442d9ee98b33a6e7a055:Formula/gcc\@6.rb | grep -v 'fails_with' > Formula/gcc\@6.rb
   echo 'Installing gcc@6 (6.4.0-2)'
   brew install gcc\@6 || brew link --overwrite gcc\@6
+  echo 'Installing check'
+  brew install check
```

Comment:
Check is not `gcc` .

Reply:
One of the tests executed in Travis CI is called `CC=gcc-6 make test-libc`